ext/snmp/snmp.c

```diff
@@ -728,19 +728,22 @@ static bool php_snmp_parse_oid(
 						zval new;
 						ZVAL_COPY_VALUE(&new, tmp_type);
 						if (!try_convert_to_string(&new)) {
```

Comment:
Why not `zval_try_get_string`? By using COPY_VALUE with a conversion you probably reach RC0 too soon.

Reply:
Yes I was planning to go back to trying that again.